geojson_pydantic/geometries.py

```diff
@@ -50,7 +65,11 @@ def _wkt_type(self) -> str:
     @property
     def wkt(self) -> str:
         """Return the Well Known Text representation."""
-        return f"{self._wkt_type}{self._wkt_inset}({self._wkt_coordinates})"
+        return self._wkt_type + (
```

Comment:
I guess this change didn't have to be in this PR, since `self.coordinates` cannot be `None` yet.